serverlessworkflow/modules/ROOT/pages/cloud/operator/add-custom-ca-to-a-workflow-pod.adoc

```diff
@@ -136,6 +136,17 @@ spec:
 
 Similar to a deployment spec, a serverless workflow has a spec.podTemplate, with minor differences, but the change is almost identical.
 In this case, we are mounting some ingress ca-bundle because we want our workflow to reach the `.apps.my-cluster-name.my-cluster-domain` SSL endpoint.
+
+In this example we pull the ingress CA of OpenShift's ingress deployment because this
+is the CA that is used to sign certificates for routes. It can be any CA that is signing the target service certificate.
+Here's how to copy the ingress ca cert to the desired namespace:
```

Comment:
```suggestion
In this example, we pull the ingress CA of OpenShift's ingress deployment because this is the CA that signs the target routes' certificates. It can be any CA that is signing the target service certificate.
Here's how to copy the ingress ca cert to the desired namespace:
```